src/main/java/com/checkmarx/ast/CxAuth.java

```diff
@@ -28,22 +28,19 @@
     private String baseuri;
     private String key;
     private String secret;
-    private String token;
+    private String apikey;
     private  URI executable = null;
     private static final Gson gson = new Gson();
 
     public CxAuth(CxScanConfig scanConfig, Logger log) throws InterruptedException, IOException, URISyntaxException {
         if(scanConfig != null) {
```

Comment:
What can they do if they pass a null scanConfig? If they must pass the scanconfig, maybe we could just have one check to see if it is null, then throw a Exception